src/libs/ajax.js

```diff
@@ -1739,6 +1739,12 @@ const Disks = signal => ({
 
   disk: (project, name) => {
     return {
+      create: ({ ...props }) => {
+        const body = jsonBody({ ...props })
+        return fetchLeo(`api/google/v1/disks/${project}/${name}`,
+          _.mergeAll([authOpts(), appIdentifier, { signal, method: 'POST' }, body])
+        )
```

Comment:
There's no need for the spread operator here. It's needlessly creating shallow copies of the object passed in.

```suggestion
      create: props => {
        const body = jsonBody(props)
        return fetchLeo(`api/google/v1/disks/${project}/${name}`,
          _.mergeAll([authOpts(), appIdentifier, { signal, method: 'POST' }, body])
        )
```
This could also be shortened a bit:
```suggestion
      create: props => fetchLeo(`api/google/v1/disks/${project}/${name}`,
        _.mergeAll([authOpts(), appIdentifier, { signal, method: 'POST' }, jsonBody(props)]))
```